public/app/plugins/datasource/loki/queryHints.ts

```diff
@@ -38,5 +42,26 @@ export function getQueryHints(query: string, series: DataFrame[]): QueryHint[] {
     }
   }
 
+  if (queryWithParser) {
+    // To keep this simple, we consider pipeline error filtering hint only is query has up to 1 parser
+    if (parserCount === 1) {
+      const hasPipelineErrorFiltering = isQueryPipelineErrorFiltering(query);
+      const hasError = extractHasErrorLabelFromDataFrame(series[0]);
+      if (hasError && !hasPipelineErrorFiltering) {
+        hints.push({
+          type: 'ADD_NO_PIPELINE_ERROR',
+          label: 'Some logs in your selected log streams might have parsing error.',
```

Comment:
Hm, I am a bit unsure about the text. Because if we add this, we know there are errors, right? Because then the `might` might be misleading. But on the other hand it is also hard to think about a better way to phrase it 😅

Reply:
Yeah, I agree. Maybe we could do `We detected that some logs in your selected log streams have parsing error.` What do you think?

Reply:
Maybe just `Some logs in your selected log streams have parsing errors.` so we get rid of the `we`?

Reply:
Yeah, I like it. Changed in https://github.com/grafana/grafana/pull/52134/commits/942594e9f69285f885632182c7f1de65d883d4c6